changelog.asciidoc

```diff
@@ -361,8 +361,8 @@ Average
 |- Added `county_code` and `party_id` fields, making it easier to determine the owner of a Session. +
 - Field `id` is changed from string to CiString, making it now case-insensitive, which had always been the idea. +
 - Changed `total_cost` field from type: `number` to `Price`, this provides the eMSP with the total cost including VAT. +
-- Renamed `start_datetime` field to `start_date_time`, to bring the naming inline with the rest of OCPI. +
-- Renamed `end_datetime` field to `end_date_time`, to bring the naming inline with the rest of OCPI. +
+- Added `start_date_time` and `end_datetime` fields. +
+- Renamed  field to `end_date_time`, to bring the naming inline with the rest of OCPI. +
```

Comment:
I would propose to make this a one liner,  "Added `start_date_time` and `end_date_time` fields"
Instead of adding and renaming in 1 changelog, the reader only sees the end result.